src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -419,6 +419,16 @@ def _do_upgrade(self):
                 'who': name_to_config_section(daemon_type),
             })
 
+        # Configure MOTD in Ceph Dashboard to notify user about the upgrade
+        # to SES7P.
+        (major, minor, patch) = self.target_version.split('.')
+        if int(major) == 15 and int(minor) >= 2 and int(patch) >= 15:
```

Comment:
Note, this is not the final version; it need to be adapted when we know which version will provide the SES7P update.